roles/aci/tasks/main.yml

```diff
@@ -9,3 +9,20 @@
   import_role:
     name: kube_proxy_and_dns
   run_once: True
+- name: include lib_openshift
+  include_role:
+    name: lib_openshift
+- name: Create namespaces
+  oc_project:
+    state: present
+    name: "{{ item }}"
+  with_items: "{{ annotate_namespace_list }}"
+  delegate_to: "{{ groups.oo_first_master.0 }}"
+- name: Get aci policy tenant name
+  shell:  "grep -m1 'aci-policy-tenant' {{ aci_deployment_yaml_file }} | awk -F '\"' '{print $4}'" 
```

Comment:
For parsing yaml files use `yedit`.  Example [provided by Vadim](https://github.com/openshift/openshift-ansible/pull/11439/files#r271328607)
```
- name: Read master storage backend setting
    yedit:
      state: list
      src: /etc/origin/master/master-config.yaml
      key: kubernetesMasterConfig.apiServerArguments.storage-backend
    register: _storage_backend
```

Reply:
Seems like there's an issue using yedit in this context, source: https://github.com/openshift/openshift-ansible/pull/11439/files#r271760006

Reply:
Thanks for looking into that.